src/modules/colorPicker/ColorPickerUI/Properties/Resources.resx

```diff
@@ -400,4 +400,8 @@
     <value>Plum</value>
     <comment>Plum color</comment>
   </data>
+  <data name="New_Label" xml:space="preserve">
+    <value>New</value>
```

Comment:
Buttons are good to initiate an action. "new" isn't really an action. What does it do?

Reply:
"Pick" is better maybe?

"Pick a color" is a bit long :(

Reply:
Changed it to 'Pick'. Let's see if we get feedback on this